action.yml

```
@@ -1,5 +1,5 @@
 name: GitHub Script
```

Do we also need 
Do we also need 
```suggestion
name: GitHub Script
author: 'GitHub'
```
here?

---

Should we unquote GitHub for consistency?